src/components/other/other/UnauthorizedHandler.tsx

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export const UnauthorizedHandler = () => {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    axios.interceptors.response.use(
+      (response) => response,
+      (error) => {
+        const statusCode = error.response ? error.response.status : null;
+        if (statusCode === 401) {
```

Comment:
So we should change it to something like HttpStatusCode.Unauthorized

Reply:
Resolved